app/controllers/location/visualization-product.js

```diff
@@ -61,7 +61,7 @@ export default Ember.Controller.extend({
   }),
   needsLegend: computed('model.visualization', function() {
     let vis = this.get('model.visualization');
-    return _.contains(['scatter', 'similarity'], vis) ? true : false;
+    return _.contains(['scatter', 'similarity', 'piescatter'], vis) ? true : false;
```

Comment:
Hahaha probably there was some more complex value there that then gone refactored out or something